cirq-core/cirq/transformers/stratify.py

```diff
@@ -48,8 +64,12 @@ def stratified_circuit(
     with operations from the same category. There is no guarantee that the
     resulting circuit will be optimally packed under this constraint.
```

Comment:
The docs description here seems super long winded, can we tighten this up ( maybe just cut it to the last paragraph or say something to the effect: "This transforms the given circuit to ensure that no operations of different categories are found in the same moment. Makes no optimality guarantees" ?

Reply:
done